test/testPartyAttachments.js

```diff
@@ -0,0 +1,1025 @@
+            localFilename: "test/images/little-boy-white.png",
+            attachmentKey: attachmentKey2,
+            resourceHref,
+          },
+          {
+            remotefileName: "profile3.png",
+            localFilename: "test/images/avatar-black.png",
+            attachmentKey: attachmentKey3,
+            resourceHref,
+          },
+        ];
+        await uploadFilesAndCheck(httpClient, filesToPut2);
+
+        // Next : try to delete one resource.
+        const response6 = await httpClient.delete({ path: attachmentUrl1 });
+        assert.equal(response6.status, 204);
+        // Now check that one attachment is gone and the others are still available
+        const response7 = await httpClient.get({ path: attachmentDownloadUrl1 });
+        assert.equal(response7.status, 404);
+        const response8 = await httpClient.get({ path: attachmentDownloadUrl2 });
+        assert.equal(response8.status, 200);
+        const response9 = await httpClient.get({ path: attachmentDownloadUrl3 });
+        assert.equal(response9.status, 200);
+      });
+
+      it("should also allow copying of existing files as (together with uploading some files) single POST multipart/form-data operation", async () => {
+        // TODO: finish this test and make it work
+        const body = {
+          type: "person",
+          name: "test user",
+          status: "active",
+        };
+        const resourceKey = uuid.v4();
+        const resourceHref = type + "/" + resourceKey;
+        const attachmentKey1 = uuid.v4();
+        const attachmentKey2 = uuid.v4();
+        const attachmentKey3 = uuid.v4();
+        const attachmentDownloadUrl1 =
+          type + "/" + resourceKey + "/attachments/profile1.png";
+
+        const response = await httpClient.put({ path: resourceHref, body });
+        assert.equal(response.status, 201);
+
+        debug("PUTting 1 profile image as attachment");
+        const filesToPut = [
+          {
+            remotefileName: "profile1.png",
+            localFilename: "test/images/orange-boy-icon.png",
+            attachmentKey: attachmentKey1,
+            resourceHref,
+          },
+        ];
+
+        await uploadFilesAndCheck(httpClient, filesToPut);
+
+        // Multiple upload with one extra attachment which is a copy of an existing attachment
+        const filesToPut2 = [
+          {
+            remotefileName: "profile2.png",
+            urlToCopy: attachmentDownloadUrl1,
+            attachmentKey: attachmentKey2,
+            resourceHref,
+          },
+          {
+            remotefileName: "profile3.png",
+            localFilename: "test/images/little-boy-white.png",
+            attachmentKey: attachmentKey3,
+            resourceHref,
+          },
+        ];
+        await uploadFilesAndCheck(httpClient, filesToPut2);
+      });
+
+      it.skip("should support handleMultipleUploadsTogether", async () => {
+        // TODO: implement this test (or remove the feature)
```

Comment:
I guess this was done here: https://github.com/katholiek-onderwijs-vlaanderen/sri4node-attachments/blob/main/test/tests.js#L163?